pihole

```diff
@@ -508,13 +508,20 @@ statusFunc() {
 }
 
 tailFunc() {
-  date=$(date +'%b %d ')
   echo -e "  ${INFO} Press Ctrl-C to exit"
-  tail -f /var/log/pihole.log | sed \
-    -e "s,\(${date}\| dnsmasq\[.*[0-9]]\),,g" \
-    -e "s,\(.*\(gravity.list\|black.list\| config \).* is \(${IPV4_ADDRESS%/*}\|${IPV6_ADDRESS:-NULL}\).*\),${COL_LIGHT_RED}&${COL_NC}," \
-    -e "s,.*\(query\[A\|DHCP\).*,${COL_NC}&${COL_NC}," \
-    -e "s,.*,${COL_DARK_GRAY}&${COL_NC},"
+
+  # Retrieve IPv4/6 addresses
+  source /etc/pihole/setupVars.conf
+
+  # Strip date from each line
+  # Colour blocklist/blacklist/wildcard entries as red
+  # Colour A/AAAA/DHCP strings as white
+  # Colour everything else as gray
+  tail -f /var/log/pihole.log | sed -E \
+    -e "s,($(date +'%b %d ')| dnsmasq[.*[0-9]]),,g" \
+    -e "s,(.*(gravity.list|black.list| config ).* is (${IPV4_ADDRESS%/*}|${IPV6_ADDRESS:-NULL}).*),${COL_RED}&${COL_NC}," \
+    -e "s,.*(query\\[A|DHCP).*,${COL_NC}&${COL_NC}," \
```

Comment:
Why are there two backslashes when there was only one before? `query\\[`

Reply:
[Shellcheck Validation for backslashes](https://github.com/koalaman/shellcheck/wiki/SC1117)